requirements/constraints.txt

```diff
@@ -240,6 +240,7 @@ typer==0.9.0
     # via python-on-whales
 typing-extensions==4.10.0
```

Comment:
Did you actually run `pip-tools` to update this pin? Which CLI options did you use? Perhaps adding something like `-P typing-extensions` would encourage it to update the pin and we'd have it matching with the new entry elsewhere.

Reply:
I did, and it propagated the version pin to the tests.

Reply:
Just ` pip-compile --allow-unsafe --output-file=requirements/test.txt --resolver=backtracking --strip-extras requirements/test.in`